lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -660,6 +661,62 @@ int amf_app_handle_pdu_session_response(
 
   return rc;
 }
+/****************************************************************************
+ **                                                                        **
+ ** Name:    convert_ambr()                                                **
+ **                                                                        **
+ ** Description: Converts the session ambr format from                     **
+ **  one defined in create_pdu_session_response to one defined in          **
+ **  pdu_session_estab_accept message                                      **
+ **                                                                        **
+ ** Inputs:  pdu_ambr_response_unit, pdu_ambr_response_value               **
+ **          ambr_unit, ambr_value                                         **
+ **                                                                        **
+ **      Return:   void                                                    **
+ **                                                                        **
+ ***************************************************************************/
+void convert_ambr(
+    uint32_t* pdu_ambr_response_unit, uint32_t* pdu_ambr_response_value,
+    uint8_t* ambr_unit, uint16_t* ambr_value) {
+  int count = 1;
+  if (*pdu_ambr_response_unit == KBPS) {
+    *pdu_ambr_response_value *= 1000;
```

Comment:
this would not work if `pdu_ambr_response_calue` is larger than MAX/1000. Also what is the contract of this function? Are you supposed to modify `pdu_ambr_response_value`? Use consts in decleration and definitions if they are not supposed to be modified.

Reply:
if unit is KBPS, you can simply start count from 2 otherwise it is at 1.

Reply:
Comment addressed.

Reply:
Comment addressed.